src/check-pr-action.ts

```diff
@@ -168,14 +168,19 @@ async function main() {
             if (requireLabel) {
                 core.setFailed(lines.join("\n"));
             }
-        } else if (change.changeType === ChangeType.TASK) {
-            lines.push(
-                "This change is marked as an *internal change* (Task), so will not be included in the changelog.",
-            );
         } else if (change.notes == null) {
-            lines.push(
-                "This change has no change notes, so will not be included in the changelog.",
-            );
+            if (change.breaking) {
+                lines.push("This change is marked as *breaking*, but is missing a changelog entry.");
+                core.setFailed(lines.join("\n"));
+            } else if (change.changeType === ChangeType.TASK) {
+                lines.push(
+                    "This change is marked as an *internal change* (Task), so will not be included in the changelog.",
+                );
```

Comment:
This looks like it might silence the warning if it is T-Task but not breaking with `Notes:` but those notes won't make it into the changelog properly will they? Looks like we have no header handling in L189-L196 for Tasks